CHANGELOG.md

```diff
@@ -65,6 +65,7 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 - [#6405](https://github.com/thanos-io/thanos/pull/6405) Index Cache: Change postings cache key to include the encoding format used so that older Thanos versions would not try to decode it during the deployment of a new version.
 - [#6432](https://github.com/thanos-io/thanos/pull/6432) Receive: Remove duplicated `gopkg.in/fsnotify.v1` dependency
 - [#6479](https://github.com/thanos-io/thanos/pull/6479) Store: *breaking :warning:* Rename `thanos_bucket_store_cached_series_fetch_duration_seconds` to `thanos_bucket_store_series_fetch_duration_seconds` and `thanos_bucket_store_cached_postings_fetch_duration_seconds` to `thanos_bucket_store_postings_fetch_duration_seconds`.
+- [#6474](https://github.com/thanos-io/thanos/pull/6474) Store/Compact: Reduce API calls against object storage when synchronizing meta files.
```

Comment:
Can we make it specific on how we reduced API calls? It is mainly replacing `Exists` call with recursive `Iter`? Mention that would be nice.

Reply:
Sure think, should be updated now.